data/tutorials/rs_00_guidelines.md

```diff
@@ -95,65 +95,68 @@ List.map f l
 > the function is meaningful.
 > 
 
-## Programming guidelines
-###  How to program
+## Programming Guidelines
+###  How to Program
 > 
 > *Always put your handiwork back on the bench,<br />
->  and then polish it and re-polish it.*
+>  then polish it and repolish it.*
```

Comment:
This is misquote from Nicolas Boileau, L'Art Poétique (Chant I.) Classic French poem from the XVIIth century:

> Hâtez-vous lentement ; et, sans perdre courage,
> Vingt fois sur le métier remettez votre ouvrage :
> Polissez-le sans cesse et le repolissez ;
> Ajoutez quelquefois, et souvent effacez.

Here is how it was translated by Soame and Dryden (reference found in Wikipedia):

> Gently make haste, of Labour not afraid;
> A hundred times consider what you've said:
> Polish, repolish, every Colour lay,
> And sometimes add; but oft'ner take away.

Most french-speaking readers of the original text: 

https://caml.inria.fr/resources/doc/guides/guidelines.fr.html

would understand the meaning, praising quality and slowness, despite the misquote. I'm a bit concerned about the rest of the world :-) What can we do?

Reply:
Oh interesting! 
How about "then polish and repeat."
or "then polish it repeatedly, making improvements with each iteration."

Reply:
I must admit I'm tempted to change the text, using the full sentence of the four verses. I'd also credit the author and the translators (and add a footnote if it was a book). I'm probably tainted by my feelings towards classic literature. 

I've failed to find a modern translation of L'Art Poétique.